### PR TITLE
[DEV BOARD] Harvey Jay M. Sison

### PR DESCRIPTION
**Pull request by Hjkun77**
I have read and verified the following upon opening this pull request to add my information to the ReactJS Philippines Dev Board:

- [x] My `avatar` is a square image
- [x] My `avatar` is below 80 kB
- [x] My `blurb` is no longer than 140 characters
- [x] I understand that only the first 5 `skills` I list will be shown on the site

**Description**
- Short description of what this PR implements or fixes

**Summary of Changes**
- Code/resources that have been added, updated or removed. Please provide rationale for changes whenever applicable

**How to Test**
- Link to dev/staging deployment and credentials whenever applicable
- Steps for performing tests
- Command line invocation to run automated tests

**Screenshots**
- Before and after screenshots whenever applicable

**Notes**
- Other things needs to be mentioned

Closes #xx - issue number

### CODE REVIEW

File: content/devs.json [hunk 1]
```diff
@@ -85,7 +85,7 @@
     "contacts": [
       {
         "type": "website",
-        "url": "https://codekcv.now.sh/"
+        "url": "http://codekcv.github.io/"
```
Comment (jromest) [inline content/devs.json:88]: Why? cc @codekcv

File: content/devs.json [hunk 2]
```diff
@@ -183,5 +189,38 @@
...
+      "ReactJS",
+      "NodeJS + Express",
+      "TypeScript",
+      "React Native",
+      "PostgreSQL",
+      "MongoDB",
+      "GatsbyJS",
+      "NextJS",      
+      "Flutter",          
+      "Django",
```
Comment (jromest) [inline content/devs.json:209]: Please remove the trailing comma.

File: content/devs.json [hunk 3]
```diff
@@ -183,5 +189,38 @@
...
+      {
+        "type": "linkedin",
+        "url": "https://linkedin.com/in/harveyjaysison/"
+      },
+      {
+        "type": "email",
+        "url": "mailto:harveyjaysison@gmail.com"
+      }
+    ]
+  },
```
Comment (jromest) [inline content/devs.json:225]: Please remove the trailing comma.

File: content/devs.json [hunk 4]
```diff
@@ -183,5 +189,38 @@
         "url": "mailto:tj.andoy@gmail.com"
       }
     ]
-  }
+  },
+  {
+    "avatar": "https://imgur.com/6b4c7ec4-251d-4319-bb24-fb1757c3598a",
```
Comment (jromest) [inline content/devs.json:194]: Can you check your avatar? I'm getting 404.

**Review by jromest — CHANGES_REQUESTED** [review]
Hey @Hjkun77, thank you for adding your profile. I have some comments and request changes 😄

**Comment by Hjkun77** [comment]
Sure I'll fix it

File: content/devs.json (hunk 1 above)
Comment (Hjkun77) [reply content/devs.json:88]: I think this part of the JSON is not mine 😅

I think i accidentally touched and made minor changes on it

File: content/devs.json (hunk 2 above)
Comment (Hjkun77) [reply content/devs.json:209]: Sure

File: content/devs.json (hunk 1 above)
Comment (codekcv) [reply content/devs.json:88]: Hello, please revert this change. It should be `https://codekcv.now.sh/`. Thanks, @jromest for noticing this.

**Comment by Hjkun77** [comment]
@jromest @codekcv I've already addressed the merge conflict too. Please do check my PR. Thank you

**Review by codekcv — CHANGES_REQUESTED** [review]
@Hjkun77 almost there! img file size should be less than 80kb.

**Comment by Hjkun77** [comment]
> @Hjkun77 almost there! img file size should be less than 80kb.

changed it! it's already 78kb

File: content/devs.json [hunk 5]
```diff
@@ -184,6 +190,39 @@
...
   },
+  {
+    "avatar": "https://i.imgur.com/LXw7ryd.jpg",
+    "name": "Harvey Jay M. Sison",
+    "title": "Platform Engineer",
+    "company": "Exora Technologies Incorporated",
+    "blurb": "Hi! 🙌🏼 I'm a full stack software engineer 💻, a UX/UI designer🎨, and an aspiring technopreneur 👨🏽‍💼  from Quezon City, Philippines",
+    "skills": [
+      "ReactJS",
+      "NodeJS + Express",
```
Comment (jromest) [inline content/devs.json:201]:
```sh
Harvey Jay M. Sison has invalid details on devs.json
[ { message:
     '"skills[1]" length must be less than or equal to 15 characters long',
    path: [ 'skills', 1 ],
    type: 'string.max',
    context:
     { limit: 15,
       value: 'NodeJS + Express',
       encoding: undefined,
       label: 'skills[1]',
       key: 1 } } ]
```

You can separate NodeJS and Express as skills.

**Review by jromest — CHANGES_REQUESTED** [review]

File: content/devs.json (hunk 5 above)
Comment (Hjkun77) [reply content/devs.json]: Separated it. Thank you

**Review by codekcv — APPROVED** [review]

**Review by jromest — APPROVED** [review]

**Pull request merged**